dygraph.js

```diff
@@ -1812,8 +1813,9 @@ Dygraph.prototype.eventToDomCoords = function(event) {
  * Returns {number} row number.
  * @private
  */
-Dygraph.prototype.findClosestRow = function(domX) {
+Dygraph.prototype.findClosestRow = function(domX, domY) {
```

Comment:
This method's docstring no longer reflects what it does. When you have `highlightMinDistance` set, would it make more sense to switch to `findClosestPoint`, rather than `findClosestRow`?